packages/switch/src/css/index.js

```diff
@@ -43,28 +46,31 @@ export default {
   [`.psds-switch__track.psds-switch__track--size-${vars.sizes.large}`]: {
     height: '24px',
     width: '48px',
-    borderRadius: '12px',
-    padding: '1px'
+    borderRadius: '12px'
+    // padding: '1px'
   },
   [`.psds-switch__track--checked.psds-switch__track--color-${vars.colors.blue}`]: {
-    backgroundColor: core.colors.blue
+    backgroundColor: core.colorsBlue.base,
+    borderColor: 'transparent'
   },
   [`.psds-switch__track--checked.psds-switch__track--color-${vars.colors.green}`]: {
-    backgroundColor: core.colors.green
+    backgroundColor: core.colorsGreen.base,
+    borderColor: 'transparent'
   },
   [`.psds-switch__track--checked.psds-switch__track--color-${vars.colors.orange}`]: {
-    backgroundColor: core.colors.orange
+    // NOTE: temp: make orange blue until we do a breaking change to remove orange
```

Comment:
noted. there is also an orange story you could comment on - or just remove

Reply:
Makes sense to me to keep the story, since it's built on the colors enum, until the colors enum changes.